packages/app/features/home/screen.tsx

```diff
@@ -30,7 +28,7 @@ export function HomeScreen() {
       <H1 sx={{ fontWeight: '800' }}>English Dictionary</H1>
       <View style={styles.container}>
         <SearchBar onButtonPress={handleSearch} />
-        {searchResults.map(result => <ResultDisplay word={result.word} phonetic={result.phonetic} />)}
+        {searchResults.map(result => <ResultDisplay word={result.word} phonetic={result.phonetic} meanings={result.meanings} />)}
```

Comment:
ditto

Think which of these is gonna be unique 🙂 
Key is just a string...

Now it's time to choose a proper unique string... maube the first element of `result.meaning`?